tests/appsec/test_blocking_addresses.py

```diff
@@ -237,7 +240,7 @@ def test_blocking_before(self):
         """Test that blocked requests are blocked before being processed"""
         # first request should not block and must set the tag in span accordingly
         assert self.set_req1.status_code == 200
-        assert self.set_req1.text == "Value tagged"
+        assert self.set_req1.text == "Value tagged" or self.set_req1.text == '"Value tagged"'
```

Comment:
In next we should respond with JSON, we can respond quoted string `"Value tagged"` but not just `Value tagged`

Reply:
Since this snippet is repeated 3 times, you could always do something like `assert self.set_req1.text.includes("Value tagged")` (not sure how you do that in python), so it matches a bit more broadly. But up to you

Reply:
```python
assert "Value tagged" in self.set_req1.text
```